src/tools/bin/radiationSyntheticDetector

```diff
@@ -52,7 +52,7 @@ parser.add_argument('--dataExtend', '-d',
                             [default=0.5 3.5 0.0 360.0]''')
 
 # extend of the data to analyze
-parser.add_argument('--analyzerExtend', '-a',
+parser.add_argument('--pluginExtend', '-a',
```

Comment:
REQUIRED:
please revert this - analyzer is the data analyzer in this tool - not a plugin